crates/rpc/eth/block.rs

```diff
@@ -70,6 +70,9 @@ impl BlockIdentifier {
             },
         }
     }
+    pub fn latest() -> BlockIdentifier {
```

Comment:
we have this too: https://github.com/lambdaclass/ethereum_rust/blob/5c791921dfda9340b144ab52aa1a9bbf90826726/crates/rpc/eth/account.rs#L38-L50

Reply:
Correct! But my idea here was having to avoid typing  `BlockIdentifier::Tag(BlockTag::Latest)` every time I wanted to instance a Latest tag.